### PR TITLE
update helm chart to upstream v1.6.8

### PR DESCRIPTION
**Pull request by glitchcrab**
This PR:

- updates the helm chart to upstream v1.6.8

### Testing

skipping testing as we don't have this app deployed anywhere yet.

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid (more info on the values schema can be found [here](https://intranet.giantswarm.io/docs/organizational-structure/teams/halo/app-updates/helm-values-schema/)).

### CODE REVIEW

**Comment by glitchcrab** [comment]
all changes were pulled in from upstream - there are no local changes.

File: helm/rook-operator/values.yaml [hunk 1]
```diff
@@ -349,7 +359,7 @@ discoverDaemonUdev:
 # Whether the OBC provisioner should watch on the operator namespace or not, if not the namespace of the cluster will be used
 enableOBCWatchOperatorNamespace: true
 
-admissionController: {}
+admissionController:
```
Comment (pipo02mix) [inline helm/rook-operator/values.yaml:362]: hmm will it not fail if we leave an object literal empty?

Comment (glitchcrab) [reply helm/rook-operator/values.yaml:362]: well i wondered that too, but this is pulled directly from the upstream 1.6.8 release https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/values.yaml#L362

:man_shrugging:

Comment (glitchcrab) [reply helm/rook-operator/values.yaml:362]: nope it looks like it should work fine this way https://github.com/giantswarm/rook-operator-app/blob/main/helm/rook-operator/templates/deployment.yaml#L269

Comment (pipo02mix) [reply helm/rook-operator/values.yaml:362]: ok I would swear I hit that in the past 🙈

**Review by pipo02mix — APPROVED** [review]

**Pull request merged**